roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -281,6 +282,8 @@
     es_cpu_limit: "{{ openshift_logging_elasticsearch_cpu_limit }}"
     es_memory_limit: "{{ openshift_logging_elasticsearch_memory_limit }}"
     es_node_selector: "{{ openshift_logging_elasticsearch_nodeselector | default({}) }}"
+    es_storage_groups: "{{ openshift_logging_elasticsearch_storage_group | default([]) }}"
+    es_container_security_context: "{{ _es_containers.elasticsearch.securityContext if _es_containers is defined and 'elasticsearch' in _es_containers and 'securityContext' in _es_containers.elasticsearch else None }}"
```

Comment:
@jcantrill  jinja2 filter `attr()` you had here previously does something else than getting a value by its key from dict. It is well described here - https://github.com/ansible/ansible/issues/24065. This is what I think does what you wanted, written as concise as possible